policy/sakuracloud_disk.rego

```diff
@@ -2,15 +2,31 @@ package main
 
 import data.exception
 import data.helpers.has_field
+import rego.v1
+
+violation_sakuracloud_disk_not_encrypted contains decision if {
```

Comment:
メッセージだけでなくresource, ruleをreturnするために `violation` を利用しました。
https://www.conftest.dev/#evaluating-policies

Reply:
- ✅ `violation rules evaluates the same as deny rules, except they support returning structured data errors instead of just strings` - violationはstructured dataを返すために使える